src/game.cpp

```diff
@@ -1559,7 +1559,7 @@ void game::rustCheck()
         int oldSkillLevel = u.skillLevel(aSkill);
 
         if (u.skillLevel(aSkill).rust(charged_bio_mem)) {
-            u.power_level -= 25;
+            u.charge_power(-25);
```

Comment:
Well, here you used the `(-25)`, why did you use `(0 - x)` in above? `(-x)` is enough.